src/scripts/visualizations/tooltip.js

```diff
@@ -137,7 +137,7 @@
 
             var dataPoints = findOriginalDataPoint(d);
 
-            this.tooltipElement.select('.text').html(getTooltipText.call(this, dataPoints[0] || d, dataPoints));
+            this.tooltipElement.select('.text').html(getTooltipText.call(this, d || dataPoints[0], dataPoints));
```

Comment:
I believe the intent is for the original data point, if found, to override the data point passed in. How about this:
```js
this.tooltipElement.select('.text').html(getTooltipText.call(this, _.isArray(dataPoints) && dataPoints[0] || d, dataPoints));
```